dashboard/app/models/user.rb

```diff
@@ -2649,4 +2655,10 @@ def code_review_groups
       errors.add(:us_state, :invalid)
     end
   end
+
+  # Values for the `cpa_compliance` attribute
+  module CpaCompliance
+    # The student's account has been approved by their parent.
+    PERMISSION_GRANTED = 'g'.freeze
```

Comment:
Any reason we went with a char here instead of a boolean?

Reply:
Because the compliance might be in multiple states: "Account created", "Permission Request sent", "Permission granted"

Reply:
Ah got it, makes sense. Maybe we can add _state or _status to the property name to make it more intuitive?

Reply:
I added `_state` to the name